docker/arm-server/Dockerfile

```diff
@@ -3,6 +3,9 @@
 # so it could execute CF instance with API in HO.
 FROM debian:stable-20211011 AS cuttlefish-arm64
 
+# to detect if it is running inside a docker
+ENV DOCKER_RUNNING=true
```

Comment:
There is code in other places (example: cvd) that checks for the existence of the `.dockerenv` file. It's probably best to make sure it exists here than to come up with another mechanism, something like `RUN touch /.dockerenv` should work.